live-ingester/src/main/resources/application.yml

```diff
@@ -69,3 +56,21 @@ hyland-experience:
       endpoint: ${hyland-experience.insight.base-url}/presigned-urls?httpMethod=POST&throwExceptionOnFailure=false
   ingester:
     endpoint: ${hyland-experience.insight.base-url}/ingestion-events?httpMethod=POST&throwExceptionOnFailure=false
+
+auth:
+  providers:
+    hyland-experience:
+      type: oauth2
+      client-id:
+      client-secret:
+      token-uri: http://localhost:8001/token
+      grant-type: urn:hyland:params:oauth:grant-type:api-credentials
+      scope: hxp.integrations
+    alfresco:
+      type: basic
+      username:
+      password:
+  retry:
+    attempts: 10
+    initial-delay: 500
+    delay-multiplier: 2
```

Comment:
I don't think this would be optimal as this is a common set of properties for 2 modules. Since we have different objects representing properties in these modules - this would produce a bit of a boilerplate code in both of them and could also add some more complexity in the integration tests.
I think the best idea would be reviewing how we handle application properties in all modules and think about some refactoring in that area. Generally, that refactoring could result in grouping authentication related properties under `alfresco` and `hyland-experience`.
But I would rather do that in a separate PR, WDYT?

Reply:
yep, good idea lets rethink that and do separately